md/profileCreation.md

```diff
@@ -0,0 +1,79 @@
+# Profile Editor
+
+## Definition
+
+Basically, a **profile** is an XML file. It contains the profile name, the mapping with the organization (which users / groups / roles / memberships have this profile) and the portal menu entries (menus accessible to a user logged in the portal with this profile).  
```

Comment:
When it's the first reference in a page, we need to use "Bonita Portal", and then "the portal" is ok.